stubs/gevent/gevent/queue.pyi

```diff
@@ -42,26 +46,25 @@ class Queue(Generic[_T]):
     def put(self, item: _T, block: bool = True, timeout: float | None = None) -> None: ...
     def put_nowait(self, item: _T) -> None: ...
     def qsize(self) -> int: ...
-    def shutdown(self, immediate: bool = False) -> None: ...
     def __bool__(self) -> bool: ...
     def __iter__(self) -> Self: ...
     def __len__(self) -> int: ...
     def __next__(self) -> _T: ...
     next = __next__
 
 @final
-class UnboundQueue(Queue[_T]):
+class UnboundQueue(SimpleQueue[_T]):
     @overload
     def __init__(self, maxsize: None = None) -> None: ...
     @overload
     def __init__(self, maxsize: None, items: Iterable[_T]) -> None: ...
     @overload
     def __init__(self, maxsize: None = None, *, items: Iterable[_T]) -> None: ...
 
-class PriorityQueue(Queue[_T]): ...
-class LifoQueue(Queue[_T]): ...
+class PriorityQueue(SimpleQueue[_T]): ...
+class LifoQueue(SimpleQueue[_T]): ...
```

Comment:
At least in 25.4.2, these derive from `Queue`: 

https://github.com/gevent/gevent/blob/df6c474a91ed70de677723747ae82a75e1c4ed3e/src/gevent/queue.py#L590